### ISSUE TITLE
[Chore] Update Side Navigation Colors

### ISSUE THREAD

**Issue by kpmurphh** [mobi/goponents#414]
According to the shell that is in tangoe.design, it appears the background color of the side navigation is the wrong color. However just changing it to the right color throws off all the other colors.

- [ ] Side navigation background color is currently #313536 - should be #202626
- [ ] Expanded dropdown background color is currently #272b2b - should be #202626 darkened 3.5%
- [ ] Hover background color is currently #202323 - should be #20626 darkened 7%
- [ ] App Drawer color should be #474747.

**Comment by grahamhency** [comment]
@kpmurphh was this changed recently? We had this discussion a while back and determined that what exists currently was the correct path forward.

**Comment by kpmurphh** [comment]
@grahamhency I know it's been a few months since I posted this issue, is there anyway this can be included in the next release? We had gone over the styles prior to me posting this issue but upon further review the colors need to be changed. Additionally (which I'll add to this issue), the app drawer color is incorrect.

**Comment by jaredami** [comment]
Hey @kpmurphh just wanted to clarify on this: am I wrong to think that making these changes to *only* the side-nav would create an inconsistency between the side-nav and the other components which use our dark-theme colors (e.g. the off-canvas, the dark themed accordions, etc.)? Just wanted to check whether introducing such inconsistency is what is intended here before moving forward on this. Thanks!

**Comment by cosmicdust73** [comment]
> Hey @kpmurphh just wanted to clarify on this: am I wrong to think that making these changes to _only_ the side-nav would create an inconsistency between the side-nav and the other components which use our dark-theme colors (e.g. the off-canvas, the dark themed accordions, etc.)? Just wanted to check whether introducing such inconsistency is what is intended here before moving forward on this. Thanks!

Hi Jared - thanks for bringing this to our attention. The intent of this ticket is certainly not to introduce inconsistency between side-nav and off-canvas. As it turns out, you are correct that the off-canvas is also not aligned to the UX team's DSM and mockups. The scope of this change will need to be updated to include the corrections to dark-theme as well.

I did a quick check, and it seems the UX team's DSM and mockups are using #202626 for both, but Goponents is using a lighter #313536 for both.  

@kpmurphh maybe we can just update the ticket to include these changes and/or identify any other color inconsistencies between side-nav and off-canvas?

**Comment by eligolding** [comment]
Can we also include dark mode for the select/dropdown component? Cuz right now rendering a select in the off canvas is not the prettiest

<img width="476" alt="Tangoe 2020-04-30 14-32-12" src="https://user-images.githubusercontent.com/5261826/80746153-684e4c00-8aef-11ea-9d4c-d4a893e0f5ca.png">

**Comment by grahamhency** [comment]
@kpmurphh, @cosmicdust73, @jaredami if that is the case, we'll definitely want to ensure all components look consistent with a dark theme. I did a quick search through our components to see which ones have some sort of dark theming enabled:
- Accordions
- Buttons
- File Upload
- Off Canvas
- Side Nav
- Tabs
- Forms (not a component, but in general labels/fieldsets etc)

We should probably look to audit these and perhaps address @eligolding's concern about the select.

**Comment by cosmicdust73** [comment]
@grahamhency @jaredami In the interest of getting the side-nav and app-drawer fixed ASAP (it is a long standing issue and looks bad), I suggest we create a separate issue for dark-theme where we can address @eligolding 's concern along with doing an audit on all of the dark-theme components.  

If possible, let's at least change the off-canvas main background color to #202626 as part of this open issue to avoid creating a glaring inconsistency.
Cc @kpmurphh

**Comment by jaredami** [comment]
@kpmurphh @cosmicdust73 @eligolding @grahamhency Thanks all for your input!

I suspected it wouldn’t be too much extra work to go ahead and make the changes necessary to update our dark theme across the board, so I went ahead and made those changes and put up a PR for them: https://github.com/mobi/goponents/pull/547

Note: this PR does *not* include the updates suggested for the `go-select`. That might be a bit more of an involved task and so I think it makes sense to do it separately if we decide to move forward with that.

One issue I did run into was that there were a few places where we were already using `#202626`, but since that color is now being used for our dark theme backgrounds, we need a darker color for the places it was previously being used. In the PR mentioned above I changed those colors to be `#20626` darkened by 7%. This darker color was applied to:

* Dark themed accordion panel borders
* Disabled dark themed form input and select borders and backgrounds
* Disabled dark themed form fieldset backgrounds

I don't believe the colors for these component variants are specified in the DSM, so I wanted to flag these so that you all can let me know whether this conforms with what is intended.

Let me know what you all think!
